### PR TITLE
Display a banner telling if guests or services are present in the conversation

### PR DESCRIPTION
**Pull request by makingthematrix**
#### APK
[Download build #11437](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11437/artifact/build/artifact/wire-dev-PR1718-11437.apk)
[Download build #11442](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11442/artifact/build/artifact/wire-dev-PR1718-11442.apk)
[Download build #11443](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11443/artifact/build/artifact/wire-dev-PR1718-11443.apk)
[Download build #11444](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11444/artifact/build/artifact/wire-dev-PR1718-11444.apk)
[Download build #11447](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11447/artifact/build/artifact/wire-dev-PR1718-11447.apk)
[Download build #11457](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11457/artifact/build/artifact/wire-dev-PR1718-11457.apk)

### CODE REVIEW

**Comment by someoneEsle** [comment]
- [ ] Bug 1
There should be no banner in a 1:1 with a bot

**Comment by someoneEsle** [comment]
- [ ] Bug 2
Once you tap the input cursor, the banner is minimized (which is correct). However, every time you type a message the banner is displayed in full again.

File: app/src/main/scala/com/waz/zclient/views/ConversationFragment.scala [hunk 1]
```diff
@@ -236,25 +236,39 @@ class ConversationFragment extends FragmentHelper {
...
-  private def openGuestsBanner(): Unit = {
-    if (!openBanner) {
-      openBanner = true
-      guestsBanner.foreach { banner =>
-        banner.setVisibility(View.VISIBLE)
-        banner.setPivotY(0.0f)
-        banner.setScaleY(1.0f)
+  private def updateGuestsBanner(hasGuest: Boolean, hasBot: Boolean, isGroup: Boolean): Unit = {
+    def openGuestsBanner(resId: Int): Unit = {
+      if (!openBanner) {
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/views/ConversationFragment.scala]: I think this variable should be called: `isBannerOpen`

**Comment by makingthematrix** [comment]
build again

**Pull request merged**